### PR TITLE
Remove six library and upgrade Python syntax

### PR DESCRIPTION
**Pull request by hugovk**
Fixes https://github.com/scrapinghub/spidermon/issues/269.

Upgrades Python syntax with https://github.com/asottile/pyupgrade using `--py36-plus`

### CODE REVIEW

File: spidermon/contrib/actions/email/ses.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
-from __future__ import absolute_import
 from boto import ses
```
Comment (Gallaecio) [inline spidermon/contrib/actions/email/ses.py:1]: GitHub shows something weird above this line for me (a line `0`, and a duplicate `session.send_raw_email(raw_message=message.as_string())` line being turned into a single one in this patch), but I checked `master` and the line before there was `from __future__ import absolute_import`, so :heavy_check_mark:

File: spidermon/contrib/actions/telegram/__init__.py [hunk 2]
```diff
@@ -1,12 +1,7 @@
-from __future__ import absolute_import
-
-import ast
```
Comment (Gallaecio) [inline spidermon/contrib/actions/telegram/__init__.py:3]: Seems unused, :heavy_check_mark:

**Review by Gallaecio — APPROVED** [review]

**Review by Gallaecio — APPROVED** [review]

**Comment by hugovk** [comment]
Merge conflict resolved!

Is there anything more I need to do for this?

File: tests/test_extension.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 try:
     import unittest.mock as mock
 except ImportError:
-    import mock
+    from unittest import mock
```
Comment (Gallaecio) [inline tests/test_extension.py:4]: Do we need this try-except? I imagine a single import should work for all supported Python 3 versions.

**Review by Gallaecio — APPROVED** [review]

**Comment by hugovk** [comment]
One year later: is there something I can do to help get this merged? It simplifies code paths and removes a dependency, so could improve installation and code speed.

Perhaps if I drop all changes other than removing the six dependency?

Otherwise I don't mind closing the PR.

Thank you!

**Review by rennerocha — APPROVED** [review]
Sorry for taking too long for reviewing this PR. It should be merged months ago!

**Comment by hugovk** [comment]
No problem, thanks! 🚀

**Pull request merged**